<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>forwarditem</class>
 <widget class="QDialog" name="forwarditem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>586</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="GraGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>280</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Gravity</string>
   </property>
   <widget class="QCheckBox" name="GAllBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ALL</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="GxxBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Gxx</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="GxyBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Gxy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="GyyBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Gyy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="GyzBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Gyz</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="GxzBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Gxz</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="GzzBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Gzz</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="GBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="MagGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>281</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Magnetic</string>
   </property>
   <widget class="QCheckBox" name="MAllBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ALL</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="HaxBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Hax</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="HayBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Hay</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="BxxBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bxx</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="BxyBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bxy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ZaBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Za</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="BxzBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bxz</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="TBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ΔT</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ByyBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Byy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="BzzBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bzz</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ByzBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Byz</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="TxBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ΔTx</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="TyBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ΔTy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="TzBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ΔTz</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="OkBtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>550</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>550</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Observation Height</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ObserHedit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="GeoMaggroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>280</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Geomagnetic field</string>
   </property>
   <widget class="QLineEdit" name="Iedit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Dedit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>I(°)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>D(°)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Gaussian Noise</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>mean</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="GaussMean">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>std</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="GaussStd">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>add proportion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AddProportion">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
